drivers/bus/ancillary_bus.c

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Software based bus for Ancillary devices
+ *
+ * Copyright (c) 2019-2020 Intel Corporation
+ *
+ * Please see Documentation/driver-api/ancillary_bus.rst for
```

Comment:
Please use new 100 characters limit in a line.
shorter the code, easier to review and merge.

Reply:
changed